todo_app/flask_config.py

```diff
@@ -1,5 +1,9 @@
 """Flask configuration class."""
 import os
+from dotenv import load_dotenv
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+load_dotenv(os.path.join(BASEDIR, '.env'))
```

Comment:
Couple of comments - I wouldn't really expect this change in the config file (particularly as it's only required for testing); it would be better to have this occur as the first part of the `test_app` fixture in each of the browser test files
Also, this strictly insists that the .env file be inside the todo_app folder which isn't where the default instructions place it. A slightly nicer way to do this is to use the `find_dotenv` function to get the relevant path which is a bit more flexible, e.g.
```
@pytest.fixture(scope='module')
def test_app():
    filepath = find_dotenv('.env')
    load_dotenv(filepath, override=True)
    ...
```